air-quality-ui/system_tests/ui/city_page.spec.ts

```diff
@@ -1,8 +1,353 @@
 import { expect, test } from '../utils/fixtures'
+import { gotoPage, setupPageWithMockData } from '../utils/helper_methods'
+import {
+  createForecastAPIResponseData,
+  createMeasurementsCityPageResponseData,
+} from '../utils/mocked_api'
 
-test('AQI snapshot assertion', async ({ cityPage }) => {
-  await cityPage.setupCityPageGraph()
-  await expect(cityPage.textFinder('Rio de Janeiro')).toBeVisible()
-  const chartShot = await cityPage.captureChartScreenshot()
-  expect(chartShot).toMatchSnapshot('rio-aqi-graph.png')
+test.use({
+  viewport: { width: 1920, height: 1080 },
+})
+
+test('vAirify logo is visible', async ({ page, banner }) => {
```

Comment:
this test should be in the banner spec I think

Reply:
done, this was a duplicate